CHANGELOG.rst

```diff
@@ -3,6 +3,33 @@
 0.55.0
 ------
 
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**🌟 New Features**
+
+- **UI**: Renku 2.0: add user pages, update group pages, update project information panel, update the search page (`#3198 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3198>`__)
+- **Search Services**: Search results return details about the creator
+  and project namespace, where before only the respective ids were
+  included
```

Comment:
nitpick: bit weird line breaking going on here compared to lines above/below? We don't really enforce any line length limits in this file, but should at least keep things consistent.